deployment/10_patch_filters.js

```diff
@@ -0,0 +1,151 @@
+/* global artifacts */
+global.web3 = web3;
+global.artifacts = artifacts;
+
+const ethers = require("ethers");
+const childProcess = require("child_process");
+
+const DappRegistry = artifacts.require("DappRegistry");
+const MultiSig = artifacts.require("MultiSigWallet");
+
+const CompoundFilter = artifacts.require("CompoundCTokenFilter");
+const ParaswapFilter = artifacts.require("ParaswapFilter");
+const UniswapV2Filter = artifacts.require("UniswapV2UniZapFilter");
+const WethFilter = artifacts.require("WethFilter");
+const ParaswapUniV2RouterFilter = artifacts.require("ParaswapUniV2RouterFilter");
+const TokenRegistry = artifacts.require("TokenRegistry");
+
+const deployManager = require("../utils/deploy-manager.js");
+const MultisigExecutor = require("../utils/multisigexecutor.js");
+
+const main = async () => {
+  const { deploymentAccount, configurator } = await deployManager.getProps();
+
+  // //////////////////////////////////
+  // Setup
+  // //////////////////////////////////
+
+  const { config } = configurator;
+  const DappRegistryWrapper = await DappRegistry.at(config.contracts.DappRegistry);
+  const TokenRegistryWrapper = await TokenRegistry.at(config.contracts.TokenRegistry);
+  const MultiSigWrapper = await MultiSig.at(config.contracts.MultiSigWallet);
+  const multisigExecutor = new MultisigExecutor(MultiSigWrapper, deploymentAccount, config.multisig.autosign);
+
+  // //////////////////////////////////
+  // Change DappREgistry ownership
+  // //////////////////////////////////
+
+  console.log("Setting the deployment account as the owner of default registry");
+  await multisigExecutor.executeCall(DappRegistryWrapper, "changeOwner", [0, deploymentAccount]);
+
+  // //////////////////////////////////
+  // Update existing filters to Argent Registry
+  // //////////////////////////////////
+
+  const filters = {};
+
+  //
+  // Compound
+  //
+  for (const [underlying, cToken] of Object.entries(config.defi.compound.markets)) {
+    if (underlying === "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359" || underlying === "0x1985365e9f78359a9B6AD760e32412f4a445E862") {
+      // remove SAI and REP filter
+      console.log(`Removing filter for Compound Underlying ${underlying}`);
+      await DappRegistryWrapper.removeDapp(0, cToken);
```

Comment:
Shouldn't we remove the corresponding filters from the `filters.CompoundFilter` array?

Reply:
🤝